front/src/containers/SearchPage/SearchPage.tsx

```diff
@@ -801,10 +808,12 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
 
     return (
       <SearchContainer>
-        <InstructionsContainer>
+        <InstructionsContainer  >
           {presearchText && (
             <Instructions>
               <RichTextEditor
```

Comment:
Do instructions even need an editor?

Reply:
Yeah they are using it to add in the presearch instructions into the page. I think they originally wanted to be able to use markup for this in particular.

Reply:
Ah I see, thanks for clarifying.